processors/base/config.py

```diff
@@ -10,14 +10,21 @@
 from dotenv import load_dotenv
 load_dotenv('.env')
 
+# Environment
+
+ENV = os.environ.get('PYTHON_ENV', 'development')
+if os.environ.get('CI'):
+    ENV = 'testing'
 
 # Storage
 
-WAREHOUSE_URL = os.environ['WAREHOUSE_URL']
-DATABASE_URL = os.environ['DATABASE_URL']
-EXPLORERDB_URL = os.environ['EXPLORERDB_URL']
-TEST_WAREHOUSE_URL = os.environ.get('TEST_WAREHOUSE_URL')
-TEST_DATABASE_URL = os.environ.get('TEST_DATABASE_URL')
+if ENV == 'testing':
+    WAREHOUSE_URL = os.environ['TEST_WAREHOUSE_URL']
+    DATABASE_URL = os.environ['TEST_DATABASE_URL']
+else:
+    WAREHOUSE_URL = os.environ['WAREHOUSE_URL']
+    DATABASE_URL = os.environ['DATABASE_URL']
+    EXPLORERDB_URL = os.environ['EXPLORERDB_URL']
```

Comment:
As the tests are always run through `tox`, and `tox` itself sets the `PYTHON_ENV = 'testing'`, I think we're safe here. But yeah, if someone runs the tests outside of `tox` for some reason, she can end up clearing the local database :sweat: